.github/workflows/ci.yaml

```diff
@@ -143,6 +143,8 @@ jobs:
                   cp -f $GITHUB_WORKSPACE/.github/workflows/ci.bazelrc .bazelrc.test
                   echo "build --config=${{ matrix.config }}" >> .bazelrc.test
                   echo "build --disk_cache=" >> .bazelrc.test
+                  bazel clean
+                  ./test.sh
```

Comment:
Running tests twice will give us the confidence that bazel clean does not hide potential state bugs.